### ISSUE TITLE
As a developer, I want to run a 508 accessibility audit on the site so that I know all visitors can access the information on the site

### ISSUE THREAD

**Issue by katherinedm-usds** [usds/justice40-tool#232]
**Description**
Run or if possible hire a resource to run a 508 audit on our site once map functionality, community list download functionality, and (at least draft) text for all supporting pages, headers, and footers have been implemented

**Tasks**
 - [x] Katherine reach out to resource recommended by VA team and learn timing, cost, IA requirements
 - [ ] Engineering (Nat?) advise if there's a version of this USDS could run internally
 - [ ] Run audit and plan any additional work to address into product roadmap

**Additional References**
- @glennettec to attach checklist from running other tool

**DoD**
 - [ ] We have a clear picture of what interactions might be limited for people with disabilities, and have alternative ways for them to get that information (via community download list, etc.)

**Comment by glennettec** [comment]
[Accessibility and Usability report from SortSite.](https://app.zenhub.com/files/358242767/bd786bc6-b7a2-45c6-a1f2-b71cd95670a2/download)

**Comment by switzersc-usds** [comment]
Creating a separate ticket for the human-centered, manual audit. This ticket will focus on the smaller fixes coming out of SortSite.

To do: 
- [ ] Create a checklist for fixes from SortSite and add to this card
- [ ] Get necessary content updates (based on recs from SortSite)

**Comment by glennettec** [comment]
Here is the SortSite report I ran this morning.

[SortSite Report - 07/14/2021](https://app.zenhub.com/files/358242767/05d19859-2a4f-49e3-a397-baece0d379f3/download)

**Comment by switzersc-usds** [comment]
@TomNUSDS to update accessibility build step to be more descriptive. 

Still waiting on more info for manual audit.

**Comment by BethMattern** [comment]
@glennettec will follow up with DHS on tooling they use (trusted tester) to see if this is something we could use

**Comment by vim-usds** [comment]
We have a cypress test that tests each page for a11y compliance
